windows/main.c

```diff
@@ -1225,6 +1226,15 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PSTR cmd, int n
                         debug("Please specify correct theme (please check user manual for list of correct values).");
                         theme = THEME_DEFAULT;
                     }
+                    argv_theme=1;
+                }
+            } else if(!wcscmp(arglist[i], L"--profile")) {
```

Comment:
Don't think this is really good idea. What if profile name contain space ? It will add more code and overhead cost as well to support profile names with spaces in case of "--profile="